core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -2744,6 +2746,12 @@ public boolean isQuietingDown() {
         return isQuietingDown;
     }
 
+    @Exported
+    @Nullable
```

Comment:
IIRC Nullable does nothing but checkfornull ensures that spotbugs forces people to check for null